src/compas_fab/robots/robot.py

```diff
@@ -1387,23 +1539,19 @@ def update(self, configuration, group=None, visual=True, collision=True):
         self.artist.update(configuration, visual, collision)
 
     def draw_visual(self):
-        """Draws the visual geometry of the robot in the respective CAD environment.
-        """
+        """Draws the visual geometry of the robot in the defined :attr:`.artist`'s CAD environment."""
```

Comment:
Grammatically, these "Draws..." docstrings are not consistent.  Should it be "Draw" instead?

Reply:
Hm, though all those would be picked up by pydocstyle..

I’ll fix!

Reply:
8616817